config.default.yaml

```diff
@@ -308,6 +308,26 @@ renewable:
       method: hydro_capacities  # 'hydro_capacities' to rescale country hydro production by using hydro_capacities, 'eia' to rescale by eia data, false for no rescaling
       year: 2013  # (optional) year of statistics used to rescale the runoff time series. When not provided, the weather year of the snapshots is used
     multiplier: 1.1  # multiplier applied after the normalization of the hydro production; default 1.0
+  csp:
+    cutout: cutout-2013-era5-tutorial
```

Comment:
This is the default configuration file; it should be "cutout-2013-era5"

Reply:
Thanks for pointing this out

Reply:
Fixed